src/core/utilities.js

```diff
@@ -94,10 +94,14 @@ class Utilities {
                 default:
                     break;
             }
-
+            console.log("Sending Message: ");
+            console.log(messageJson);
             ws.send(JSON.stringify(messageJson));
-            if (secondMessageJson)
+            if (secondMessageJson){
+                console.log("Sending Second Message: ");
+                console.log(secondMessageJson);
```

Comment:
Change it to  console.log("Sending Second Message: ", secondMessageJson);

Reply:
Edit this one too (for the readability of the log messages).

Reply:
done. please check